### PR TITLE
feat: refactor example pages

### LINKED ISSUE #1887: [FEATURE] refactor examples and pages to contain real source code in Storybook
**Describe how this feature will help us improve**
Describe why do you think this feature is needed. 

Currently when we display code of examples and pages in the Storybook it is completely not usable. 
image

I would recommend refactoring stories.js of these components to include all the source code that was used to create this componet (basically copy paste component structure to the stories.js instead of using `template: '<Category/>`.

**Is your proposal related to a problem?**
Describe the issue or type its number

**Describe the solution you'd considered**
A concise description of your preferred solution. Things to address include:

  - Details of the technical implementation
  - Tradeoffs made in design decisions
  - Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.

**Additional context**
Anything you would like to add but was not sure which section it fits.

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #1887 

# Scope of work
Move example pages code into stories to show it in storybook.  

# Screenshots of visual changes
image

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
@Baroshem Do we also need to show in docs all the props needed to render example pages or just showing just code in story tab and in docs? 
image

In docs tab you can find rendered code not the source code of our example so it cannont be copied and used. Take a look:
image
 
But in story tab everything is fine:
image

So what do you think, is it okay or should it be fixed somehow?

**Comment by Baroshem** [comment]
@AdamPawlinski 
Please rebase your branch to the latest changes in 0.10.9. It should improve the CI actions ;)

File: packages/vue/.storybook/main.js [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
-  addons: [
-    "@storybook/addon-essentials",
-    "@storybook/addon-links",
-    "@storybook/addon-a11y",
-    "@storybook/preset-scss",
-    "@storybook/addon-storysource", 
-  ],
-}
\ No newline at end of file
+	stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
```
Comment (justyna-13) [inline packages/vue/.storybook/main.js:7]: Please take a look at indentations in files :) there should be 2 spaces instead of 4

**Comment by justyna-13** [comment]
Can you add this also to the form and cart? :) 
Screenshot from 2021-07-12 12-06-33

**Comment by AdamPawlinski** [comment]
Yes of course. It should be added.

**Comment by AdamPawlinski** [comment]
The cypress test on login page didn't pass because this page was changed. Now I fixed it and adjusted test.

**Comment by AdamPawlinski** [comment]
Fixes were made @justyna-13  please see if it's okay now.

File: packages/vue/src/examples/pages/login/Login.vue [hunk 2]
```diff
@@ -1,114 +1,128 @@
...
-      </div>
-    </transition>
-  </SfModal>
+        </div>
+      </transition>
+    </SfModal>
+    <SfButton
+      style="margin: 0 auto"
+      data-testid="open-modal-button"
+      @click="openModal = true"
```
Comment (Baroshem) [inline packages/vue/src/examples/pages/login/Login.vue:121]:
```suggestion
      @click="toggleModal"
```

File: packages/vue/src/examples/pages/login/Login.vue [hunk 3]
```diff
@@ -153,6 +168,11 @@ export default {
       this.lastName = "";
     },
   },
+  methods: {
+    toggleModal(event) {
+      this.openModal = event;
+    },
```
Comment (Baroshem) [inline packages/vue/src/examples/pages/login/Login.vue:174]:
```suggestion
    toggleModal() {
      this.openModal = !this.openModal;
    },
```

File: packages/vue/src/examples/pages/login/Login.vue [hunk 4]
```diff
@@ -1,114 +1,128 @@
...
-          or Log In To Your Account
-        </SfButton>
-      </div>
-    </transition>
-  </SfModal>
+        </div>
+      </transition>
+    </SfModal>
+    <SfButton
+      style="margin: 0 auto"
```
Comment (Baroshem) [inline packages/vue/src/examples/pages/login/Login.vue:119]: Let's create a class out of it. This is supposed to be code example that someone can copy and paste to their project so it should not contain inline styles.

File: packages/vue/src/stories/Websites.stories.mdx [hunk 5]
```diff
@@ -11,7 +11,7 @@ import { Meta } from "@storybook/addon-docs/blocks";
 
 # Who is using Storefront UI?
 
-Checkout these webistes built with Storefront UI!
+Checkout these websites built with Storefront UI!
```
Comment (Baroshem) [inline packages/vue/src/stories/Websites.stories.mdx:14]: 👍

**Review by Baroshem — CHANGES_REQUESTED** [review]

File: packages/vue/src/examples/pages/login/Login.vue (hunk 4 above)
Comment (AdamPawlinski) [reply packages/vue/src/examples/pages/login/Login.vue:119]: Okay I did.

File: packages/vue/src/examples/pages/login/Login.vue (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/src/examples/pages/login/Login.vue]: changed

**Review by Baroshem — APPROVED** [review]

**Review by justyna-13 — COMMENTED** [review]
Only two things and the rest looks great :) 
1. Cart story is deleted and again there is a problem with docs view with modal 
Screenshot from 2021-07-16 08-58-41


2. I think it'd be good to have SfHeader only on the top. It looks weird when it is on the source code
Screenshot from 2021-07-16 08-59-27

**Comment by AdamPawlinski** [comment]
Cart page and SfHeader in default fixed.

**Comment by Baroshem** [comment]
@justyna-13 please check once again and if it is fine, please approve so that Adam can merge it

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**